basics_of_logging_in_sst-core/Example02/ExampleComponent.h

```diff
@@ -0,0 +1,65 @@
+#ifndef MY_COMPONENT_H
+#define MY_COMPONENT_H
+
+#include <sst/core/component.h>
+#include <sst/core/output.h>
+#include <sst/core/params.h>
+#include <cstdarg>
+#include "Logger.h"
+
+namespace Example02
+{
+    // This is a very simple component.  It only registers a clock and 
+    // prints log messages as the clock handler is called.
+    //
+    // Remember, all components inherit from SST::Component
+    //
+    class ExampleComponent : public SST::Component
+    {
+        public:
+          
+            // Constructor/Destructor
+            //
+            ExampleComponent(SST::ComponentId_t id, SST::Params &params);
+            ~ExampleComponent() {}
+
+            // Standard SST::Component functions.  These all need to
+            // be implemented in the component, even if they are empty.
+            //
+            void setup(void);
+            void finish(void);
+
+            // Clock handler.  This is the method called from the clock event.
```

Comment:
May want to comment on what the return bool represents.  It represents the answer to the question: Do you want to be removed from the clock list?